Plugins/Flow.Launcher.Plugin.Explorer/ViewModels/SettingsViewModel.cs

```diff
@@ -9,27 +9,20 @@ namespace Flow.Launcher.Plugin.Explorer.ViewModels
 {
     public class SettingsViewModel
     {
-        private readonly PluginJsonStorage<Settings> storage;
-
         internal Settings Settings { get; set; }
 
         internal PluginInitContext Context { get; set; }
 
-        public SettingsViewModel(PluginInitContext context)
+        public SettingsViewModel(PluginInitContext context, Settings settings)
         {
             Context = context;
-            storage = new PluginJsonStorage<Settings>();
-            Settings = storage.Load();
+            Settings = settings;
         }
 
-        public Task LoadStorage()
-        {
-            return Task.Run(() => Settings = storage.Load());
-        }
 
         public void Save()
         {
-            storage.Save();
+            Context.API.SaveSettingJsonStorage<Settings>();
         }
```

Comment:
one more question, why does this have to call the save directly?

Reply:
I guess it is the design of Explorer plugin. I didn't change it. It contains a call for save every time a new item is edited.
It is not ISavable, so I believe it won't be called twice. However, do you think we should remove that?

Reply:
oh right, it's not ISavable, just calls infra's save directly. Nah don't worry about it for this PR, using the API is at least better. Let's get this merged in.